addons/account/tests/test_portal_attachment.py

```diff
@@ -275,6 +275,8 @@ def test_01_portal_attachment(self):
         )
         self.assertEqual(res.status_code, 200)
         self.out_invoice.invalidate_recordset(['message_ids'])
-        self.assertEqual(len(self.out_invoice.message_ids), 3)
-        self.assertEqual(self.out_invoice.message_ids[0].body, "<p>test message 3</p>")
-        self.assertEqual(len(self.out_invoice.message_ids[0].attachment_ids), 1)
+        # Only messages from the current user not OdooBot
+        messages = self.out_invoice.message_ids.filtered(lambda msg: msg.author_id == self.partner_a)
+        self.assertEqual(len(messages), 3)
+        self.assertEqual(messages[0].body, "<p>test message 3</p>")
+        self.assertEqual(len(messages[0].attachment_ids), 1)
```

Comment:
Why is the test changed? Where the extra message (from odoo bot) coming from compared to before the PR?

Reply:
Shouldn't the message_post related stuff be gone from diff already when rebasing on the already merged message post PR?